packages/nextjs/scaffold.config.ts

```diff
@@ -24,6 +24,10 @@ export const bgChainL2 = {
     public: { http: ["https://rpc.l2.buidlguidl.com"] },
     default: { http: ["https://rpc.l2.buidlguidl.com"] },
   },
+  blockExplorers: {
+    etherscan: { name: "BG L2 Explorer", url: "https://explorer.l2.buidlguidl.com" },
+    default: { name: "BG L2 Explorer", url: "https://explorer.l2.buidlguidl.com" },
```

Comment:
Why do you need this two times?

Reply:
Actually `wagmi` docs did the same thing in https://wagmi.sh/react/chains#build-your-own

![Screenshot 2023-08-31 at 9 43 04 PM](https://github.com/BuidlGuidl/l2.buidlguidl.com/assets/80153681/9f11afd2-14b5-476a-a0c0-5e8d074e9b53)

So I copied and pasted it from there, I think we can remove `etherscan` but also maybe we can keep it just in case wagmi / viem uses it or in the future we use some library that uses it to show more info, just to be safe 😅 ? 

Also, our block explorer endpoint is the same as etherscan so we won't break anything 🙌